.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,26 @@
+Review CI configuration
+Review our semgrep.yml config file.
```

Comment:
Are these comments? If so, shouldn’t they be commented lines?

Reply:
True. Editing.